event-internal.h

```diff
@@ -205,6 +206,31 @@ struct event_once {
 	void *arg;
 };
 
+#define EVENT_WATCHER_PREPARE_TYPE 0
+#define EVENT_WATCHER_CHECK_TYPE   1
+#define EVENT_WATCHER_TYPES        2
```

Comment:
Can we name it something like `EVENT_WATCHES_TYPE_MAX` or something similar, it makes meaning of this variable cleaner (that this is just the last of the watches defines)

Reply:
also can you come up with more compact name, for example how about `EVW_PREPARE` (but this is just an advice and maybe it is too compact)